install.js

```diff
@@ -113,7 +113,7 @@ whichDeferred.promise
     if (customDownloadUrl) {
       downloadUrl = customDownloadUrl
     } else if (process.platform === 'linux' && process.arch === 'x64') {
-      downloadUrl += 'linux-x86_64.zip'
+      downloadUrl += 'u1404-x86_64.zip'
```

Comment:
this is really an ubuntu 14.04 targeted release, so I would understand needing a better if check here.